### ISSUE TITLE
Broken links in unit test docs

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8579]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)| @scop reported|
|-|-|
|Trac ID|trac#8579|
|Type|enhancement|
|Created|2016-07-09 10:59:57Z|

<details><summary>Searchable metadata</summary>

```
trac-id__8579 8579
type__enhancement enhancement
reporter__scop scop
priority__low low
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1468061997139624 1468061997139624
changetime__1474198569965514 1474198569965514
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)<a name="note_1"></a>|@scop commented|
|-|-|

https://github.com/twisted/twisted/pull/305

**Comment by twisted-trac** [comment]
|<img alt="Ville Skyttä's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|Ville Skyttä commented|
|-|-|

In changeset 0fdbf1f92a6a26ab972798ff284f0d69236c8a31
```
#!CommitTicketReference repository="" revision="0fdbf1f92a6a26ab972798ff284f0d69236c8a31"
Add topfile for #8579
```

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 6c1ffd86d14e3dcd8e2298451b38b01933776e6b
```
#!CommitTicketReference repository="" revision="6c1ffd86d14e3dcd8e2298451b38b01933776e6b"
Merge pull request #307 from twisted/8579-scop-unittest-links

Author: scop
Reviewer: rodrigc
Fixes: #8579

Fix broken links in unit test docs
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing the review tag from closed tickets
